core/src/test/java/nl/nn/adapterframework/extensions/graphviz/GraphvizEngineTest.java

```diff
@@ -40,12 +40,14 @@
 	public void canInitDefaultWithoutErrors() throws IOException {
 		GraphvizEngine engine = new GraphvizEngine();
 		assertNotNull(engine);
+		engine.close();
```

Comment:
Dit zou ik in een @after method doen

Reply:
Dan moet ik de engine in een global field zetten....